.github/ISSUE_TEMPLATE/bug_report_v9_Accordion_permit.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This requires a new file per each component, right? This does not scale and don't have proper automation behind it (new components will not have these files).

Is there need to use templates? Why not use a single template as JS string and use [`body`](https://github.com/sindresorhus/new-github-issue-url#body) field?

Reply:
> This requires a new file per each component, right? This does not scale

Unfortunately, correct. Every new component will need to have this file created as well as a label on GitHub. 
We need these files to enable our partners who don't have write access to the repository, to create issues which are labelled with the component name. Labels are used to show the list of known issues at the end of component page and I think it'll streamline the bug fixing process in general. We could also achieve this by creating a GitHub bot to add these labels after the user creates the issue, but I don't think it's a better solution, because it's harder to maintain.

> This doesn't have proper automation behind

Maybe we could deal with scaffolding in another PR?

> Why not use a single template as JS string and use [body](https://github.com/sindresorhus/new-github-issue-url#body) field?

We actually use the body field [here](https://github.com/microsoft/fluentui/pull/22189/files#diff-65842b512b296b4a85ee6466ec2edd0760707c0d72653e86a0d2e8d7bccccbe5R42) and set it to the contents of [bug_report_v9.md](https://github.com/microsoft/fluentui/pull/22189/files#diff-8e6ae5baaa11905cc0f95c776a7361e00da1c7b0223311392b9d6c1cafcf0f46). We need to use these template files as well to specify which labels will be assigned to the issue.